# Maize TF ChIP-Seq/RNA-Seq papers

Author: Ji Huang

Date: 2019-03-18

Last modified date: 2022-06-06

I documented maize ChIP-Seq or RNA-Seq dataset I can find, including **TF** and **Histone marks**.

## TFs

1. [KN1][1]. [GRMZM2G017087][9]. [RNA-Seq](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38487). [ChIP-Seq](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE39161)

2. [FEA4][2]. [Zm00001d037317][10]. [RNA-Seq + ChIP-Seq](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE61954).

3. [RA1][3]. [GRMZM2G003927][11]. [RNA-Seq + ChIP-Seq](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE51050)

4. [O2][4]. [GRMZM2G015534][12]. [RNA-Seq](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE61830). [ChIP-Seq](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE63991).

5. [O11][5]. [Zm00001d003677][13]. [RNA-Seq](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE97422). [ChIP-Seq](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE102051).

6. [GIF1][6]. [Zm00001d033905][14]. [RNA-Seq + ChIP-Seq](https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP131943). Immature tassel.

7. [AN3][7]. This gene is the same as GIF1, except a different name. [ChIP-Seq](https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=ERP106997).

8. [P1][8]. [GRMZM2G084799][15]. [RNA-Seq pericarp](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38413). [RNA-Seq silk](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38414). [ChIP-Seq](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38587).

9. [CCA1][19]. [GRMZM2G014902][20]. [Zm00001d049543](https://www.maizegdb.org/gene_center/gene/Zm00001d049543). [ChIP-Seq](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67655). Aerial tissues.

10. [bZIP4][21]. [Zm00001d018178][22]. The processed ChIP data is on [figureshare][23], but no raw data could be found on SRA.

11. [zmm28 ][24]. [Zm00001d022088_P002][25]. ChIP-Seq and RNA-Seq data at [SRP166189][26]
12. [tb1 - teosinte branched1][27]. [Zm00001d033673](https://www.maizegdb.org/gene_center/gene/Zm00001d033673). ChIP-Seq and RNA-Seq data at [SRP182814][28]. Tiller buds.
13. [gt1 - grassy tillers1][27]. [Zm00001d028129](https://www.maizegdb.org/gene_center/gene/Zm00001d028129). ChIP-Seq and RNA-Seq data at [SRP182814][28]. Tiller buds.
14. [ns1 - narrow sheath1](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA633509). [Zm00001d004889](https://www.maizegdb.org/gene_center/gene/Zm00001d004889). ChIP-Seq and RNA-Seq data at [SRP262137](https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP262137&o=acc_s%3Aa). I didn't find the published paper.
15. [ZmM16 ChIP-seq using ear primordia tissue](https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP272724). I didn't find the published paper. Maybe [Zm00001d042618](https://www.maizegdb.org/gene_center/gene/Zm00001d042618)
16. [ZmABI19](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8136913/). [Zm00001d011712](https://www.maizegdb.org/gene_center/gene/Zm00001d011712). ChIP-Seq and RNA-Seq data in seeds at [SRP259231](https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP259231)
17. [dlf1 - delayed flowering1](https://nph.onlinelibrary.wiley.com/doi/10.1111/nph.16772). [Zm00001d022613](https://www.maizegdb.org/gene_center/gene/Zm00001d022613). ChIP-Seq and RNA-Seq data in SAM at [PRJNA616067](https://www.ncbi.nlm.nih.gov//bioproject/PRJNA616067)

## Epigenomes

1. Shoot and Root: H3K4me3, H3K9ac, H3K27me3, H3K36me3, smallRNA, Methylation and mRNA. [Genome-Wide and Organ-Specific Landscapes of Epigenetic Modifications and Their Relationships to mRNA and Small RNA Transcriptomes in Maize, 2009 Plant Cell](http://www.plantcell.org/content/21/4/1053). GEO:[GSE15286](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE15286).

2. Shoot and Root from B73, Mo17, hybrid: H3K4me3, H3K9ac, H3K36me3, smallRNA, Methylation and mRNA. [Conservation and divergence of transcriptomic and epigenomic variation in maize hybrids, 2013 Genome Biology](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2013-14-6-r57). GEO: [GSE43142](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE43142)

3. B73 ear BS-Seq, H3K27me2 and H3K9me2. [Accessible DNA and Relative Depletion of H3K9me2 at Maize Loci Undergoing RNA-Directed DNA Methylation, 2014 Plant Cell](http://www.plantcell.org/content/26/12/4903). SRP: [SRP047420](https://www.ncbi.nlm.nih.gov//bioproject/PRJNA261802)

4. B73 seedling H3K9me2. [Genomic Distribution of H3K9me2 and DNA Methylation in a Maize Genome, 2014 PloS One](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4133378/). SRP: [SRP043372](https://www.ncbi.nlm.nih.gov/sra/?term=SRP043372)

5. B73 seedling third leaf H3K4me3. [RNA-directed DNA methylation enforces boundaries between heterochromatin and euchromatin in the maize genome, 2015 PNAS](https://www.pnas.org/content/112/47/14728.long). SRA: [SRX1073672](https://www.ncbi.nlm.nih.gov/sra/SRX1073672/)

6. B73 seed H4K5ac. [Genome-Wide Mapping of Targets of Maize Histone Deacetylase HDA101 Reveals Its Function and Regulatory Mechanism during Seed Development, 2016 Plant Cell](http://www.plantcell.org/content/28/3/629). SRA: [SRP067369](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA305809).

7. B73/Mo17 hybrid endosperm H3K27me3. [Genome-wide high resolution parental-specific DNA and histone methylation maps uncover patterns of imprinting regulation in maize, 2014 Genome Research](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3875858/). SRA: [SRP011991 ](https://www.ncbi.nlm.nih.gov/sra?term=SRP011991).

8. B73 husk V2 H3K9ac. [Genome-wide mapping of transcriptional enhancer candidates using DNA and chromatin features in maize, 2017 Genome Biology](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1273-4). GEO: [GSE94251](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE94251). DNaseI-Seq: [GSE94291](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE94291).

9. B73 leaf protoplast BS-Seq, mRNA, *in situ* Hi-C, ATAC-Seq, H3K4me3, H3K27me3, H3K27ac. [3D Chromatin Architecture of Large Plant Genomes Determined by Local A/B Compartments, 2017 Molecular Plant](https://www.sciencedirect.com/science/article/pii/S1674205217303398?via%3Dihub). SRA: [PRJNA391551](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA391551/).

10. Long-range interaction, B73 seedling shoot. [Chromatin interaction maps reveal genetic regulation for quantitative traits in maize](https://www.nature.com/articles/s41467-019-10602-5#Sec9). [Long-range interactions between proximal and distal regulatory regions in maize](https://www.nature.com/articles/s41467-019-10603-4#Sec11).

11. Stable unmethylated DNA regions in maize (also rice, barley, sorghum and brachy), and maize root ATAC-Seq. [Stable unmethylated DNA demarcates expressed genes and their cis-regulatory space in plant genomes](https://www.pnas.org/content/117/38/23991). The maize UMR bed file is **Supplemental Dataset S3**. The rice UMR bed file is **Supplemental Dataset S9**. Download from [here](https://www.pnas.org/content/suppl/2020/09/02/2010250117.DCSupplemental). Genome version: Zma-AGPv4;  H. vulgare L-Ensemble_v42; S. bicolor v3.1.1-Phytozome12;  O. sativa v7.0-Phytozome11;  B.distachyon v3.1-Phytozome12.

12. A valuable resource including RNA-seq, Hi-C, ATAC-seq, ChIP-seq (for H3K4me3, H3K9ac and H3K27me3) and Bisulfite-seq was generated for mapping chromatin interactions and epigenome states in maize ear and tassel (B73). [NCBI-PRJNA599454](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA599454). [Paper-3D genome architecture coordinates trans and cis regulation of differentially expressed ear and tassel genes in maize, 2020, Genome Biology](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02063-7#availability-of-data-and-materials)

13. One MethylC-seq, six RNA-seq, 30 ChIP-seq, 37 DAP-seq, six ATAC-seq, one Hi-C, two HiChIP, and three STARR-seq samples of maize (B73). [NCBI-GSE120304](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE120304). [Paper-Widespread long-range cis-regulatory elements in the maize genome, 2019, NP](https://www.nature.com/articles/s41477-019-0547-0)

14. Protoplast 104 TF ChIP Seq and epigenome. [Paper-Reconstructing the maize leaf regulatory network using ChIP-seq data of 104 transcription factors, 2020, NC](https://www.nature.com/articles/s41467-020-18832-8) .ChIP peaks at [GSE137972](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE137972). The ChIP data is in ConnecTF also. For ATAC, H3K27me3, H3K9ac, Pol2 ChIP, data at [SRP183225](https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP183225&o=assay_type_s%3Aa%3Bacc_s%3Aa). Input controls: [SRR12321692](https://trace.ncbi.nlm.nih.gov/Traces/sra?run=SRR12321692) and [SRR12321693](https://trace.ncbi.nlm.nih.gov/Traces/sra?run=SRR12321693)

15. Maize MOA-Seq in ear. [Paper-The native cistrome and sequence motif families of the maize ear, 2021, PloS Genetics](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1009689). Data is mapped to B73v3 and B73v5 and is available in the Supplementary material.

## Other methods:

1. Maize [ARF DAP-Seq][16]. This paper described **14** ZmARFs using DAP-Seq. [GEO](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE111857). It also include four tissue ATAC-Seq data, but I didn't find them on SRA/GEO.
2. [DAP-Seq Cell paper][17] described **ZmARF29**, but I'm not sure what gene it is.
3. [PlantTFDB][18] has some predicted TF binding sites projected by orthology.


## RNA-Seq

1. [fascicled ear1](https://www.pnas.org/content/118/7/e2019218118). Not cloned yet.


[1]:http://www.genesdev.org/cgi/pmidlookup?view=long&pmid=22855831
[2]:http://www.plantcell.org/content/27/1/104.long
[3]:https://genome.cshlp.org/content/24/3/431.long
[4]:http://www.plantcell.org/content/27/3/532.long
[5]:http://www.plantcell.org/content/30/2/375.long
[6]:http://www.plantcell.org/content/30/2/360
[7]:http://www.plantphysiol.org/content/177/2/447
[8]:http://www.plantcell.org/content/24/7/2745.long
[9]:https://www.maizegdb.org/gene_center/gene/kn1
[10]:https://www.maizegdb.org/gene_center/gene/fea4
[11]:https://www.maizegdb.org/gene_center/gene/12577
[12]:https://www.maizegdb.org/gene_center/gene/12495
[13]:https://www.maizegdb.org/gene_center/gene/12492
[14]:https://www.maizegdb.org/gene_center/gene/gif1
[15]:https://www.maizegdb.org/gene_center/gene/p1
[16]:https://www.nature.com/articles/s41467-018-06977-6
[17]:https://www.cell.com/cell/fulltext/S0092-8674(16)30481-0#secsectitle0015
[18]:http://planttfdb.cbi.pku.edu.cn/index.php?sp=Zma
[19]:https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006197#sec016
[20]:https://www.maizegdb.org/gene_center/gene/GRMZM2G014902
[21]: http://www.plantphysiol.org/content/178/2/753.long#sec-16
[22]: https://www.maizegdb.org/gene_center/gene/bzip4
[23]: https://figshare.com/articles/ZmbZIP4-ChIPseq-input-1_wiggle_tar_gz/6225806/2
[24]: https://www.pnas.org/content/early/2019/10/29/1902593116
[25]: https://www.maizegdb.org/gene_center/gene/Zm00001d022088
[26]: https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP166189
[27]: https://www.nature.com/articles/s41467-019-11774-w
[28]: https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP182814




